pkg/healthcheck/healthcheck.go

```diff
@@ -1787,6 +1800,35 @@ func (hc *HealthChecker) checkCanGet(namespace, group, version, resource string)
 	return hc.checkCanPerformAction("get", namespace, group, version, resource)
 }
 
+func (hc *HealthChecker) checkAPIService(serviceName string) error {
+	apiServiceClient, err := apiregistrationv1client.NewForConfig(hc.kubeAPI.Config)
+	if err != nil {
+		return err
+	}
+
+	apiStatus, err := apiServiceClient.APIServices().Get(serviceName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	available := false
+	var errorMessage string
+	for _, condition := range apiStatus.Status.Conditions {
+		if condition.Type == "Available" {
+			available = true
+			if condition.Status == "True" {
+				return nil
+			}
+			errorMessage = fmt.Sprintf("%s: %s", condition.Reason, condition.Message)
```

Comment:
If we just return this error immediately here, we don't need the `errorMessage` or `available` variables.

Reply:
@adleong I was under the impression that we need to check all conditions as there might be multiple available conditions. See https://github.com/linkerd/linkerd2/pull/3689#discussion_r364840296 @droidnoob, thats correct, right?

Reply:
@zaharidichev I was under that impression. If there's only going to be one `available` condition then we can return right away.

Reply:
Yes I was also under that impression after the comment here https://github.com/linkerd/linkerd2/pull/3689#discussion_r364840296. @adleong yeah ignore that. We can just return right away. Makes total sense